karma/makeconf.js

```diff
@@ -73,7 +73,9 @@ function makeConf(defaultFile, browserNoActivityTimeout, requires) {
           prefs: {
             'media.gstreamer.enabled': false,
             'media.navigator.permission.disabled': true,
-            'media.navigator.streams.fake': true
+            'media.navigator.streams.fake': true,
+            'media.autoplay.enabled.user-gestures-needed': false,
```

Comment:
Do users start experiencing this issue on FF66? Do we need to mention it somewhere or handle it the way Unified-plan is handled?

Reply:
@syerrapragada We should update the documentation/README. I'll create a ticket for it.